core/tests/protractor_utils/SkillEditorPage.js

```diff
@@ -171,19 +167,18 @@ var SkillEditorPage = function() {
     await this.selectDifficultyForRubric(difficulty);
     var editRubricExplanationButtons = element.all(
       by.css('.protractor-test-edit-rubric-explanation-' + difficulty));
-    await waitFor.elementToBeClickable(
-      await editRubricExplanationButtons.get(explIndex),
-      'Edit Rubric Explanation button takes too long to be clickable');
-    await editRubricExplanationButtons.get(explIndex).click();
+    await action.click(
+      'Edit RubricExplanation Buttons',
+      editRubricExplanationButtons.get(explIndex));
     var editor = element(
       by.css('.protractor-test-rubric-explanation-text'));
     await waitFor.visibilityOf(
       editor, 'Rubric explanation editor takes too long to appear');
-    await (await browser.switchTo().activeElement()).sendKeys(explanation);
-    await waitFor.elementToBeClickable(
-      saveRubricExplanationButton,
-      'Save Rubric Explanation button takes too long to be clickable');
-    await saveRubricExplanationButton.click();
+    await action.sendKeys(
+      'browser switchTo Active Element', (
+        await browser.switchTo().activeElement()), explanation);
+    await action.click(
+      'Save Rubric Explanation Button', saveRubricExplanationButton);
   };
 
   this.expectRubricExplanationsToMatch = async function(
```

Comment:
Can we check that `rubricExplanationsForDifficulty.get(i)` is visible before calling `.getText()` on it below?